packages/mongodb-rag-ingest/src/commands/embed.ts

```diff
@@ -60,6 +67,12 @@ const commandModule: CommandModule<
 
 export default commandModule;
 
+export const doInitEmbedCommand = async ({ embeddedContentStore }: ResolvedConfig) => {
+  if (embeddedContentStore) {
+    await embeddedContentStore?.init?.();
+  }
```

Comment:
Should we throw an error if the embedded content store isn't defined?

Reply:
i think we should